megamek/i18n/megamek/client/messages_de.properties

```diff
@@ -369,11 +434,76 @@ CommonSettingsDialog.showMapHexPopup=Tooltips verwenden.
+CommonSettingsDialog.gameSummaryMM.tooltip=At the end of certain phases, save an image of the minimap to {0}, creating a visual summary of the progression of the game.
+CommonSettingsDialog.generateNames=Generate a random name for the pilots of new units.
+CommonSettingsDialog.guiScale=GUI-Skalierung
+CommonSettingsDialog.guiScaleTT=Adjusts the size of UI elements. Experimental - currently only affects some UI elements.
+CommonSettingsDialog.mouseWheelZoom=Zoom der Karte mit dem Mausrad
+CommonSettingsDialog.mouseWheelZoomFlip=Zoomrichtung f�r Mausrad umkehren
+CommonSettingsDialog.mapScrollText=Clicking on the Minimap and using the numpad arrow keys always\nscrolls the main display. Additionally, you may choose to scroll:
+CommonSettingsDialog.nagForUnJamRAC=Confirm done when movement is declared with jammed RAC (or AC with (Unofficial) Unjam Ultra Autocannons).
+CommonSettingsDialog.nagForWiGELanding=Confirm when WiGE does not move far enough to remain airborne.
+CommonSettingsDialog.ReportKeywords=Schl�sselbegriffe f�r Reports
+CommonSettingsDialog.scrollSensitivity=Scroll sensitivity:
+CommonSettingsDialog.tooltipDismissDelay=Tooltip dismiss delay:
+CommonSettingsDialog.tooltipDismissDelayTooltip=Sets the delay before a tooltip is dismissed. -1 uses the default for your look and feel.
+CommonSettingsDialog.tooltipDistSuppressionTooltip=<html>To prevent tooltips from following the cursor on the board, the tooltip will be suppressed if the mouse moves more than this distance between updates. <br>The default value is the diagonal of a single hex.</html>
+CommonSettingsDialog.tooltipDistSuppression=Tooltip distance suppression:
+CommonSettingsDialog.showWpsinTT=Zeige Waffen im Einheiten-Tooltip
+CommonSettingsDialog.showArmorMiniVisTT=Show mini armor visualization in the tooltip.
+CommonSettingsDialog.showPilotPortraitTT=Show pilot portrait in tooltip.
+CommonSettingsDialog.AOHexSHadows=Show ambient-occlusion-like hex shadows at the base of hills
+CommonSettingsDialog.useShadowMap=Schatten f�r Terrain und Geb�ude
+CommonSettingsDialog.useInclines=Hex-Abschr�gungen an Level�nderungen
+CommonSettingsDialog.levelHighlight=Level-Hervorhebung (verdickte Hexkanten bei Level�nderungen)
+CommonSettingsDialog.floatingIso=Floating isometric (no hex sides in isometric mode)
+CommonSettingsDialog.mmSymbol=StratOps Einheiten-Symbole auf der Minimap
+CommonSettingsDialog.useSoftCenter=Sanfte Kartenbewegung, wenn zu einer anderen Einheit gewechselt wird
+CommonSettingsDialog.useSoftCenterTip=Instead of an instant snap, the view is smoothly transitioned
+CommonSettingsDialog.showIPAddressesInChat=Zeige IP-Adressen im Chat (ACHTUNG: Sicherheitsrelevant)
+CommonSettingsDialog.showIPAddressesInChat.tooltip=<html>If enabled, all server and client IP addresses will be logged in the public chat box.<br>Enabling this can allow information disclosure, such as private IP addresses of the server.</html>
```

Comment:
```suggestion
CommonSettingsDialog.showIPAddressesInChat.tooltip=<html>If enabled, all server and client IP addresses will be logged in the public chat box. <br>Enabling this can allow information disclosure, such as private IP addresses of the server.</html>
```

Reply:
This is also English